### PR TITLE
Feat/develop/coin base integration

### PR DESCRIPTION
**Pull request by ShowriSrinivas**
# Changes

1. Coin base integration

# Screenshots

image
image

### CODE REVIEW

File: app/src/component/CheckoutComponent.tsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ export const CheckoutComponent: React.FC<CheckoutProps> = ({
 
   return (
     <MojitoCheckout
+      userInfo={{
+        email: 'showri.srinivas@ionixxtech.com',
```
Comment (alexinx) [inline app/src/component/CheckoutComponent.tsx:23]: can you add playground Input UI

File: app/src/layout/CheckoutLayout.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ const paymentOptions: CheckboxOptions[] = [
   { field: 'walletconnect', label: 'Wallet Connect', checked: true },
   { field: 'wire', label: 'Wire Transfer', checked: true },
   { field: 'creditcard', label: 'Credit Card', checked: true },
+  { field: 'coinbase', label: 'Coinbase', checked: true },
```
Comment (alexinx) [inline app/src/layout/CheckoutLayout.tsx:65]: can you plz update. doc & arch?

File: app/src/component/CheckoutComponent.tsx (hunk 1 above)
Comment (ShowriSrinivas) [reply app/src/component/CheckoutComponent.tsx:23]: Changed to auth0

File: app/src/layout/CheckoutLayout.tsx (hunk 2 above)
Comment (ShowriSrinivas) [reply app/src/layout/CheckoutLayout.tsx:65]: added

**Pull request merged**